mobile/src/main/java/com/alexstyl/specialdates/search/ContactWithEventsSearch.java

```diff
@@ -28,13 +28,13 @@ public static ContactWithEventsSearch newInstance(Context context) {
         return new ContactWithEventsSearch(context, contactProvider, nameMatcher);
     }
 
-    ContactWithEventsSearch(Context context, ContactProvider contactProvider, NameMatcher nameMatcher) {
+    private ContactWithEventsSearch(Context context, ContactProvider contactProvider, NameMatcher nameMatcher) {
```

Comment:
That is a very good point. Back when this PR was created there were no tests around that so I felt like it was okay to do. 
One and a half month later I am working on the search bit once again and this time I am writing tests around it. This is how the class looks like now: https://github.com/alexstyl/Memento-Calendar/blob/all_events/mobile/src/main/java/com/alexstyl/specialdates/search/ContactWithEventsSearch.java#L17

I could swear that I did this in order to write some tests but I cannot find the branch for that. Maybe I forgot to push 🤔